sysinfo doesn't exist outside of Linux unlike sysconf.

_studio/shared/umc/core/vm/src/vm_sys_info_linux32.c:28:10: fatal error: 'sys/sysinfo.h' file not found
#include <sys/sysinfo.h>
         ^~~~~~~~~~~~~~~

--- _studio/shared/umc/core/vm/src/vm_sys_info_linux32.c.orig	2019-10-30 19:56:10 UTC
+++ _studio/shared/umc/core/vm/src/vm_sys_info_linux32.c
@@ -25,8 +25,6 @@
 #include <sys/utsname.h>
 #include <unistd.h>
 
-#include <sys/sysinfo.h>
-
 uint32_t vm_sys_info_get_cpu_num(void)
 {
 #if defined(ANDROID)
